---
type: LLMFrontend
pgmo:
  horizon: 15.0
  d_graph_resolution: 2.5
  output_mesh_resolution: 0.005
min_object_vertices: 40
prune_mesh_indices: true
gvd:
  max_distance_m: 4.5
  min_diff_m: 0.1
  min_basis_for_extraction: 1
  voronoi_config:
    min_distance_m: 0.30
    parent_l1_separation: 20
    parent_cos_angle_separation: 0.2
graph_extractor:
  type: CompressionGraphExtractor
  compression_distance_m: 1.0
  min_node_distance_m: 0.4
  min_edge_distance_m: 0.25
  node_merge_distance_m: 0.5
  add_heuristic_edges: false
places_clustering:
  type: PlaceClustering
  clustering:
    type: $(arg places_clustering_type)
    stop_value: 0.0
    tasks:
      type: RosTaskEmbeddings
    norm:
      type: cosine
    merge:
      type: weighted_mean
  min_assocation_iou: 0.8
